src/Transform/ONNX/ConstProp.cpp

```diff
@@ -109,6 +110,13 @@ bool isVariadicOperandFromDenseONNXConstantOp(ValueRange operands) {
   return llvm::all_of(operands, [](Value v) { return isDenseONNXConstant(v); });
 }
 
+Value ConstZeroTensor(
+    PatternRewriter &rewriter, Location loc, ShapedType type) {
+  return createONNXConstantOpWithDenseAttr(rewriter, loc,
+      DenseElementsAttr::get(
+          type, rewriter.getZeroAttr(type.getElementType())));
+}
```

Comment:
Since we are moving to using DialectBuilder, It would be better to use OnnxBuilder for ONNXConstantOp here.

Reply:
ok, changed in PR #2140